### DISCUSSION TITLE
No green light in the graph

### DISCUSSION THREAD

**Discussion by iamnizamul** [community/community#67123] [New to GitHub]
I've created 2 repositories having around 15 commits each
But it doesn't show in graph also shows no contributions.
There was a different mail in git and github.
I made them the same.
What next to be done to sort it out?

**Comment by raphaelscholle** [comment]
looks like the issue is resolved, the graph isn't updated every second but in intervals :


![image](https://github.com/community/community/assets/68374617/6da37b5c-68ef-4532-b9c1-c78118e0ea0d)
